static/gsApp/utils/billing.tsx

```diff
@@ -546,6 +547,12 @@ export function getBestActionToIncreaseEventLimits(
   const isPaidPlan = subscription.planDetails?.price > 0;
   const hasBillingPerms = organization.access?.includes('org:billing');
 
+  // Managed subscriptions should always request (can't self-serve via checkout)
+  // These organizations manage their billing through partners (Heroku, Vercel, GitHub, etc.)
+  if (isDisabledByPartner(subscription)) {
+    return UsageAction.REQUEST_ADD_EVENTS;
+  }
```

Comment:
same as before, self-serve partners are able to use checkout, as well as some channel partnerships